### ISSUE TITLE
I have been using the magic wand ai to do changes in pictures and removing people and more. 

### ISSUE THREAD

**Issue by apicazo98** [photopea/photopea#6375]
Just having an inquiry of when did credits where added. I return from some vacations and now everytime I use the magic wand it takes me 3 credits from what I have during the month. Last year when I was using it it did not require credits. Is this something new or is it just me?

**Comment by camoody1** [comment]
Photopea does not have a credit system like you're describing. In order to use the AI tools without a Premium account, the user is asked to watch one or two advertisements. Are you using Photopea directly from the Photopea.com website?

**Comment by apicazo98** [comment]
> Photopea does not have a credit system like you're describing. In order to use the AI tools without a Premium account, the user is asked to watch one or two advertisements. Are you using Photopea directly from the Photopea.com website?

I have the premium for one month and while using the magic replace it shows me "3 credits has been used, you now hace 1983 credits to use during the month"

**Comment by photopea** [comment]
Each Premium user has 3000 AI credits per month. Select - "Remove BG" costs 1 credit. The Magic Replace costs 3 credits. So you can use the Magic Replace at most 1000x a month. The message is displayed only after you spend 1000 credits in that month. 99% of our Premium users spend less than 1000 AI credits in a month.

I know that you did not "sign up for this". Please, tell me your email address inside Photopea and I will fully refund you.

**Comment by photopea** [comment]
We have a user, who used the Magic Replace 6000 times in the last week (maybe it is you). 70% of the Magic Replace usage is done by 1% of Premium users.

**Comment by photopea** [comment]
@apicazo98 The message is not displayed anymore, because our system just switched to February, and you have 3000 AI credits for February. The message will appear again if you spend more than 1000 credits during February.

**Comment by camoody1** [comment]
> We have a user, who used the Magic Replace 6000 times in the last week (maybe it is you). 70% of the Magic Replace usage is done by 1% of Premium users.

WOW! I think the credit system is a totally acceptable compromise for the cost. Maybe you should apply a version of this plan to non-premium users, also. Make them create an account to use the AI functions and then give them credits that they can purchase more of using a PayPal account. Maybe $1 gets them 100 credits. Might spur more people to buy a premium account. 🤷🏽 Maintain the advertisements for people who don't want to pay money, but still give them a monthly credit limit since it's so costly to you.

**Comment by apicazo98** [comment]
Don’t worry about it. I am fine I was just curious since it didn’t appear before. The service is so perfect so I will still be using it a lot daily if not I will search more assistance by how to get more credits.On 31 Jan 2024, at 17:06, Photopea ***@***.***> wrote:﻿
@apicazo98 The message is not displayed anymore, because our system just switched to February, and you have 3000 AI credits for February. The message will appear again if you spend more than 1000 credits during February.

**Comment by photopea** [comment]
We introduced it two days ago. You spent 1000 AI credits in two days, so 3000 per month will probably not be enough for you :(

**Comment by photopea** [comment]
Just to be clear: you can still use the Magic Replace if you run out of AI credits, you will just have to watch video ads.

**Comment by pavmcgav** [comment]
Hello, Is there a way to sign up with a different email to get more ai credits? I tried it but my ai credits update so I guess you're doing it based on IP or device or whatever info you obtain. 

Please advise as I made a 2nd account to get more ai credits but it seems like that's not the way it works?

Thank youo

**Comment by photopea** [comment]
Hi, did you write me an email? It should be fixed now.

**Comment by pavmcgav** [comment]
> Hi, did you write me an email? It should be fixed now.

Yes it's fixed now thank you for such a prompt fix! That was excellent!

**Comment by JMullet261993** [comment]
im a premium user. i alredy used my ai credits. I want more but cannot purchase more as it says im already a preimum member. please either add an unlimited ai credit opition or remedy how i can get more 3k isnt enogh for me. im doint this for myself as i rarly play games or anything else

**Comment by pavmcgav** [comment]
> im a premium user. i alredy used my ai credits. I want more but cannot purchase more as it says im already a preimum member. please either add an unlimited ai credit opition or remedy how i can get more 3k isnt enogh for me. im doint this for myself as i rarly play games or anything else

Hello, you can email support about this. I signed up again with a different email to get more credits after I ran out with my frist account. At first my credits didn't reset for the new account, but I emailed support and they remedied the situation. 

support@photopea.com
